src/els_config.erl

```diff
@@ -118,10 +118,10 @@ do_initialize(RootUri, Capabilities, {ConfigPath, Config}) ->
   ok = set(code_lenses    , CodeLenses),
   ok = set(diagnostics    , Diagnostics),
   %% All (including subdirs) paths used to search files with file:path_open/3
+  ProjectDirs = els_utils:resolve_paths( [[ RootPath ]], RootPath, true),
+
   ok = set( search_paths
-          , lists:append([ project_paths(RootPath, AppsDirs, true)
-                         , project_paths(RootPath, DepsDirs, true)
```

Comment:
@dgud @alanz Are dependencies guaranteed to be within the root path? If they are not, we may have a problem here, since we are not searching any longer among dependencies.

Reply:
Confirmed as a major issue which breaks the latest master. This should be fixed in #732.